tests/unit/llms/providers/test_openai.py

```diff
@@ -72,7 +89,7 @@ def provider_config_with_specific_parameters():
     )
 
 
-def test_basic_interface(provider_config):
+def test_basic_interface(provider_config, fake_certifi_store):
```

Comment:
Not sure how we are using the fake_certifi_store in this test - or - why we need it . @kobutton